### ISSUE TITLE
pip install notifiers fails with FileNotFoundError

### ISSUE THREAD

**Issue by pcp1976** [notifiers/notifiers#308]
**Describe the bug**
Attempting to install notifiers via pip results in a file not found error - raised whilst attempting to access README.MD from setup.py:

```
command: /usr/local/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-pv6vufd1/notifiers/setup.py'"'"'; __file__='"'"'/tmp/pip-install-pv6vufd1/notifiers/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base pip-egg-info
         cwd: /tmp/pip-install-pv6vufd1/notifiers/
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-pv6vufd1/notifiers/setup.py", line 5, in <module>
        long_description = Path("README.MD").read_text()
      File "/usr/local/lib/python3.7/pathlib.py", line 1206, in read_text
        with self.open(mode='r', encoding=encoding, errors=errors) as f:
      File "/usr/local/lib/python3.7/pathlib.py", line 1193, in open
        opener=self._opener)
      File "/usr/local/lib/python3.7/pathlib.py", line 1046, in _opener
        return self._accessor.open(self, flags, mode)
    FileNotFoundError: [Errno 2] No such file or directory: 'README.MD'
```

**To Reproduce**

Attempt to install notitifiers using pip

**Expected behavior**
That notifiers should successfully install

**Additional context**
Python version: 3.7
OS: Alpine (during build of docker container)

**Note**

I checked inside the archive for notifiers on pypi, and there is no README.MD in the same directory as setup.py.

**Comment by liiight** [comment]
hmm, without docker seems to work:

```

Welcome to Termux!

Wiki:            https://wiki.termux.com
Community forum: https://termux.com/community
IRC channel:     #termux on freenode
Gitter chat:     https://gitter.im/termux/termux
Mailing list:    termux+subscribe@groups.io

Search packages:   pkg search <query>
Install a package: pkg install <package>
Upgrade packages:  pkg upgrade
Learn more:        pkg help
$ python -V
Python 3.7.4
$ python -m pip install notifiers
Collecting notifiers
  Using cached https://files.pythonhosted.org/pa
ckages/06/ee/e3a3ba875a4464046ab51ceaacc5d28386c
09e8e6b773e31cd5a08d94da5/notifiers-1.2.0-py3-no
ne-any.whl
Requirement already satisfied: click>=7.0 in /da
ta/data/com.termux/files/usr/lib/python3.7/site-
packages (from notifiers) (7.0)
Requirement already satisfied: jsonschema>=3.0.0
 in /data/data/com.termux/files/usr/lib/python3.
7/site-packages (from notifiers) (3.0.1)
Requirement already satisfied: requests>=2.21.0 
in /data/data/com.termux/files/usr/lib/python3.7
/site-packages (from notifiers) (2.21.0)
Requirement already satisfied: rfc3987>=1.3.8 in
 /data/data/com.termux/files/usr/lib/python3.7/s
ite-packages (from notifiers) (1.3.8)
Requirement already satisfied: pyrsistent>=0.14.
0 in /data/data/com.termux/files/usr/lib/python3
.7/site-packages (from jsonschema>=3.0.0->notifi
ers) (0.14.11)
Requirement already satisfied: six>=1.11.0 in /d
ata/data/com.termux/files/usr/lib/python3.7/site
-packages (from jsonschema>=3.0.0->notifiers) (1
.12.0)
Requirement already satisfied: setuptools in /da
ta/data/com.termux/files/usr/lib/python3.7/site-
packages (from jsonschema>=3.0.0->notifiers) (40
.8.0)
Requirement already satisfied: attrs>=17.4.0 in 
/data/data/com.termux/files/usr/lib/python3.7/si
te-packages (from jsonschema>=3.0.0->notifiers) 
(19.1.0)
Requirement already satisfied: urllib3<1.25,>=1.
21.1 in /data/data/com.termux/files/usr/lib/pyth
on3.7/site-packages (from requests>=2.21.0->noti
fiers) (1.24.2)
Requirement already satisfied: chardet<3.1.0,>=3
.0.2 in /data/data/com.termux/files/usr/lib/pyth
on3.7/site-packages (from requests>=2.21.0->noti
fiers) (3.0.4)
Requirement already satisfied: idna<2.9,>=2.5 in
 /data/data/com.termux/files/usr/lib/python3.7/s
ite-packages (from requests>=2.21.0->notifiers) 
(2.7)
Requirement already satisfied: certifi>=2017.4.1
7 in /data/data/com.termux/files/usr/lib/python3
.7/site-packages (from requests>=2.21.0->notifie
rs) (2018.4.16)
Installing collected packages: notifiers
Successfully installed notifiers-1.2.0
You are using pip version 19.0.3, however versio
n 19.2.3 is available.
You should consider upgrading via the 'pip insta
ll --upgrade pip' command.
$
```
I'm on vacation currently but I'll try to test via docker in the upcoming week.

**Comment by exequiel09** [comment]
@liiight this needs to be done first before #78 can be accomplished. My brew script errors out because of this.

**Comment by liiight** [comment]
Found the issue I believe, pushing a fix ASAP.

**Comment by liiight** [comment]
just released 1.2.1 with the fix, please try again

**Comment by pcp1976** [comment]
Nice job, thanks liiight.

**Comment by liiight** [comment]
sure, thanks for the report. It's an interesting issue, since this does not feel any part of the pypi released flow even though I tried to include a file that was not part of the source.

I should drop the old school packaging and move to flit or poetry asap... 🤷‍♂

**Comment by pcp1976** [comment]
My pleasure.

Here's what I now see (for confirmation):

```
vagrant@contrib-jessie:/server_data/megatron/touch$ docker run -it python:3.7-alpine sh
Unable to find image 'python:3.7-alpine' locally
3.7-alpine: Pulling from library/python
9d48c3bd43c5: Already exists
c0ea575d71b9: Pull complete
0f535eceebd5: Pull complete
8a30f5893bea: Pull complete
c1d30ace7b67: Pull complete
Digest: sha256:9363cb46e52894a22ba87ebec0845d30f4c27efd6b907705ba9a27192b45e797
Status: Downloaded newer image for python:3.7-alpine
/ # pip install notifiers
Collecting notifiers
  Downloading https://files.pythonhosted.org/packages/cf/e0/f6a6cefe31ebf03df6db87511c559e8ad6eb2f933c99aba00232e8e802f2/notifiers-1.2.1-py3-none-any.whl (49kB)
     |████████████████████████████████| 51kB 1.8MB/s
Collecting jsonschema>=3.0.0 (from notifiers)
  Downloading https://files.pythonhosted.org/packages/54/48/f5f11003ceddcd4ad292d4d9b5677588e9169eef41f88e38b2888e7ec6c4/jsonschema-3.0.2-py2.py3-none-any.whl (54kB)
     |████████████████████████████████| 61kB 4.0MB/s
Collecting click>=7.0 (from notifiers)
  Downloading https://files.pythonhosted.org/packages/fa/37/45185cb5abbc30d7257104c434fe0b07e5a195a6847506c074527aa599ec/Click-7.0-py2.py3-none-any.whl (81kB)
     |████████████████████████████████| 81kB 4.0MB/s
Collecting rfc3987>=1.3.8 (from notifiers)
  Downloading https://files.pythonhosted.org/packages/65/d4/f7407c3d15d5ac779c3dd34fbbc6ea2090f77bd7dd12f207ccf881551208/rfc3987-1.3.8-py2.py3-none-any.whl
Collecting requests>=2.21.0 (from notifiers)
  Downloading https://files.pythonhosted.org/packages/51/bd/23c926cd341ea6b7dd0b2a00aba99ae0f828be89d72b2190f27c11d4b7fb/requests-2.22.0-py2.py3-none-any.whl (57kB)
     |████████████████████████████████| 61kB 3.2MB/s
Collecting pyrsistent>=0.14.0 (from jsonschema>=3.0.0->notifiers)
  Downloading https://files.pythonhosted.org/packages/b9/66/b2638d96a2d128b168d0dba60fdc77b7800a9b4a5340cefcc5fc4eae6295/pyrsistent-0.15.4.tar.gz (107kB)
     |████████████████████████████████| 112kB 3.4MB/s
Collecting attrs>=17.4.0 (from jsonschema>=3.0.0->notifiers)
  Downloading https://files.pythonhosted.org/packages/6b/e8/2ecaf86b128a34e225807f03b22664302937ab826bd3b7eccab6754d29ea/attrs-19.2.0-py2.py3-none-any.whl (40kB)
     |████████████████████████████████| 40kB 5.3MB/s
Collecting six>=1.11.0 (from jsonschema>=3.0.0->notifiers)
  Downloading https://files.pythonhosted.org/packages/73/fb/00a976f728d0d1fecfe898238ce23f502a721c0ac0ecfedb80e0d88c64e9/six-1.12.0-py2.py3-none-any.whl
Requirement already satisfied: setuptools in /usr/local/lib/python3.7/site-packages (from jsonschema>=3.0.0->notifiers) (41.2.0)
Collecting urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 (from requests>=2.21.0->notifiers)
  Downloading https://files.pythonhosted.org/packages/e0/da/55f51ea951e1b7c63a579c09dd7db825bb730ec1fe9c0180fc77bfb31448/urllib3-1.25.6-py2.py3-none-any.whl (125kB)
     |████████████████████████████████| 133kB 3.1MB/s
Collecting chardet<3.1.0,>=3.0.2 (from requests>=2.21.0->notifiers)
  Downloading https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl (133kB)
     |████████████████████████████████| 143kB 2.5MB/s
Collecting idna<2.9,>=2.5 (from requests>=2.21.0->notifiers)
  Downloading https://files.pythonhosted.org/packages/14/2c/cd551d81dbe15200be1cf41cd03869a46fe7226e7450af7a6545bfc474c9/idna-2.8-py2.py3-none-any.whl (58kB)
     |████████████████████████████████| 61kB 3.6MB/s
Collecting certifi>=2017.4.17 (from requests>=2.21.0->notifiers)
  Downloading https://files.pythonhosted.org/packages/18/b0/8146a4f8dd402f60744fa380bc73ca47303cccf8b9190fd16a827281eac2/certifi-2019.9.11-py2.py3-none-any.whl (154kB)
     |████████████████████████████████| 163kB 2.6MB/s
Building wheels for collected packages: pyrsistent
  Building wheel for pyrsistent (setup.py) ... done
  Created wheel for pyrsistent: filename=pyrsistent-0.15.4-cp37-cp37m-linux_x86_64.whl size=56384 sha256=ba07690125ed8d3f91d8a6ec115ac8676360cb39cb23fe81ccb82e7c5223759e
  Stored in directory: /root/.cache/pip/wheels/bb/46/00/6d471ef0b813e3621f0abe6cb723c20d529d39a061de3f7c51
Successfully built pyrsistent
Installing collected packages: six, pyrsistent, attrs, jsonschema, click, rfc3987, urllib3, chardet, idna, certifi, requests, notifiers
Successfully installed attrs-19.2.0 certifi-2019.9.11 chardet-3.0.4 click-7.0 idna-2.8 jsonschema-3.0.2 notifiers-1.2.1 pyrsistent-0.15.4 requests-2.22.0 rfc3987-1.3.8 six-1.12.0 urllib3-1.25.6
```

All good 👍 

WRT poetry - I'm still getting in to pakaging with pypi, most of what I do lives in docker images and ends up being part of the source for the container. I guess I will at some point want to use pypi "in anger" but haven't yet needed to.

**Comment by exequiel09** [comment]
@liiight  i think you forgot to push the tags for v1.2.1? There is no entry in the GH releases tab.
